app/src/main/java/com/github/se/gomeet/ui/mainscreens/profile/EditProfile.kt

```diff
@@ -244,85 +279,60 @@ fun EditProfile(
 
                 TextField(
                     value = firstName.value,
-                    onValueChange = { newValue ->
-                      if (newValue.isNotBlank() && newValue.isNotEmpty()) {
-                        firstName.value = newValue
-                      }
-                    },
+                    onValueChange = { newValue -> firstName.value = newValue },
                     label = { Text("First Name") },
                     singleLine = true,
                     modifier = Modifier.fillMaxWidth(),
                     colors = textFieldColors)
-
+                if (!firstNameValid && !firstClick) {
+                  Text(text = "First Name is not valid", color = Color.Red)
+                }
                 Spacer(modifier = Modifier.size(16.dp))
-
+                if (!lastNameValid && !firstClick) {
```

Comment:
This should probably be under the last name field (for now, this text appears above the last name field, but all the other error messages appear under their respective fields)

Reply:
You're right, thanks for the feedback ! This has been fixed